server/jetstream.go

```diff
@@ -331,34 +347,35 @@ func (a *Account) EnableJetStream(limits *JetStreamAccountLimits) error {
 			metafile := path.Join(tdir, fi.Name(), JetStreamMetaFile)
 			metasum := path.Join(tdir, fi.Name(), JetStreamMetaFileSum)
 			buf, err := ioutil.ReadFile(metafile)
+			errEvent := jsStartUpError.Detail(JSErrorDetail{"metafile": metafile, "metasum": metasum})
 			if err != nil {
-				s.Warnf("  Error reading StreamTemplate metafile %q: %v", metasum, err)
+				a.sendJetStreamErrorAdvisory(s, errEvent.UserMessage("Error reading StreamTemplate metafile").Wrap(err))
```

Comment:
these of course fire events super early like before clients are likely to be there.  But if you make a stream that catch them and it gets loaded early it will catch these so its quite good (assuming its not the one failing :))

We might consider sorting the order streams are loaded in so its deterministic and one can arrange for a audit stream to exist first.